includes/wikia/models/DesignSystemGlobalFooterModelV2.class.php

```diff
@@ -268,6 +268,15 @@ public function getData() {
 						'href' => 'https://www.dndbeyond.com/',
 						'tracking_label' => 'explore.dnd-beyond',
 					],
+					[
+						'type' => 'link-text',
+						'title' => [
+							'type' => 'text',
+							'value' => 'Cortex RPG'
+						],
+						'href' => 'https://www.cortexrpg.com/',
+						'tracking_label' => 'explore.dnd-beyond',
```

Comment:
Needs updating

Reply:
good catch. copy/paste and then proof read. 😉